openedx/core/djangoapps/edx_global_analytics/README.md

```diff
@@ -0,0 +1,88 @@
+# edX global analytics
+
+`EdX global analytics` is part of edx-platform as Django application. It has arisen out of [Open edX](https://open.edx.org)
+requirement to collect edx-platform installations statistics data from all over the world.
+
+So one at 24 hours this application fetches and sends edX installation statistics to [Acceptor](https://github.com/raccoongang/acceptor) developed by @raccoongang team.
```

Comment:
you need to add yourself to authors file of edx-platform as well (See edx docs).